src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -105,11 +126,29 @@ public static IServiceCollection AddUmbracoConfiguration(this IServiceCollection
         {
             if (configuration == null) throw new ArgumentNullException(nameof(configuration));
 
-            var configsFactory = new AspNetCoreConfigsFactory(configuration);
+            services.Configure<ActiveDirectorySettings>(configuration.GetSection(Constants.Configuration.ConfigPrefix + "ActiveDirectory"));
+            services.Configure<ConnectionStrings>(configuration.GetSection("ConnectionStrings"), o => o.BindNonPublicProperties = true);
+            services.Configure<ContentSettings>(configuration.GetSection(Constants.Configuration.ConfigPrefix + "Content"));
+            services.Configure<CoreDebugSettings>(configuration.GetSection(Constants.Configuration.ConfigPrefix + "Core:Debug"));
+            services.Configure<ExceptionFilterSettings>(configuration.GetSection(Constants.Configuration.ConfigPrefix + "ExceptionFilter"));
+            services.Configure<GlobalSettings>(configuration.GetSection(Constants.Configuration.ConfigPrefix + "Global"));
```

Comment:
Some of these constants have the prefixes already on them (just noticed it in my branch and started adding to them)

Reply:
Thanks, I only found a problem with SecuritySettings..  What else do you think is a problem?

Reply:
No problem, just seen the constants in `Constants-Configuration.cs` and wondered if they could be combined. Not exactly urgent, but I'd just been looking at it for the healthchecks

Reply:
All the constants are prefixes (Ending with ":"). As a rule of thumb, we want constants we the same hardcoded value is used multiple tiles. This is only the case with the prefixes :)